tmc-prototype/CentralNode/CentralNode/CentralNode.py

```diff
@@ -251,99 +174,98 @@ def push_event(self, evt):
     # ---------------
     # General methods
     # ---------------
-
     def init_device(self):
-        SKABaseDevice.init_device(self)
         # PROTECTED REGION ID(CentralNode.init_device) ENABLED START #
+        """ This method initializes the attributes and properties"""
+        SKABaseDevice.init_device(self)
         try:
             # To read forwarded attributes
-            #print "Subarray 1 Health:", self.subarray1HealthState
-            #print "Subarray 2 Health:",self.subarray2HealthState.get_x()
+            # print "Subarray 1 Health:", self.subarray1HealthState
+            # print "Subarray 2 Health:",self.subarray2HealthState.get_x()
 
-            self._subarray1_health_state = 0
-            self._subarray2_health_state = 0
+            self._subarray1_health_state = CONST.ENUM_OK
+            self._subarray2_health_state = CONST.ENUM_OK
 
-            self.set_state(PyTango.DevState.ON)
+            self.set_state(tango.DevState.ON)
             # Initialise Properties
-            self.SkaLevel = 1
+            self.SkaLevel = CONST.INT_SKA_LEVEL
 
             # Initialise Attributes
-            self._health_state = 0
+            self._health_state = CONST.ENUM_OK
             self._admin_mode = 0
-            self._telescope_health_state = 0
+            self._telescope_health_state = CONST.ENUM_OK
             self.subarrayHealthStateMap = {}
             self._dish_leaf_node_devices = []
             self._leaf_device_proxy = []
 
-        except Exception as e:
-            print "Unexpected error on initialising properties and attributes on Central Node device."
-            self._read_activity_message = "Unexpected error on initialising properties and attributes on Central Node device."
-            self.devlogmsg("Unexpected error in initialising properties and attributes on Central Node device.", 2)
-            self._read_activity_message = "Error message is: \n" + str(e)
-            print "Error message is: \n", e
+
+        except Exception as except_occured:
+            print CONST.ERR_INIT_PROP_ATTR_CN
+            self._read_activity_message = CONST.ERR_INIT_PROP_ATTR_CN
+            self.devlogmsg(CONST.ERR_INIT_PROP_ATTR_CN, 2)
```

Comment:
Same. Check similar appearances elsewhere.

Reply:
Replaced magic numbers with respective TANGO Log Levels in Central Node code. Need to check in the same in other control nodes and Dish Master.

Reply:
Replaced magic numbers with respective TANGO Log Levels in SubarrayNode, DishLeafNode, and DishMaster. Commit id is 8138492.